### PR TITLE
Update TwG Setup UI - No publication yet scenario

### PR DESCRIPTION
**Pull request by nfmohit**
## Summary

Addresses issue:

- https://github.com/google/site-kit-wp/issues/5528#issuecomment-1192386429

## Relevant technical choices

This PR updates the TwG Setup UI - No publication yet scenario screen so that:
- The Thank with Google icon/logo size matches the Figma design.

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/thank-with-google/components/setup/SetupMain.js [hunk 1]
```diff
@@ -101,8 +101,8 @@ export default function SetupMain( { finishSetup } ) {
 							<div className="googlesitekit-setup-module__heading">
 								<div className="googlesitekit-setup-module__logo">
 									<ThankWithGoogleIcon
-										width="33"
-										height="33"
+										width="40"
+										height="39"
```
Comment (aaemnnosttv) [inline assets/js/modules/thank-with-google/components/setup/SetupMain.js:105]: Comparing with others, it looks like 33x33 is correct. Why would we make it larger?

image
image

Currently on develop:
image

**Review by aaemnnosttv — COMMENTED** [review]
@nfmohit perhaps I'm missing something but it looks like what we have now is correct. Can you clarify?

**Comment by nfmohit** [comment]
> @nfmohit perhaps I'm missing something but it looks like what we have now is correct. Can you clarify?

Whoops, my bad. I hadn't checked the module icon sizes for the other modules. I made the change following the Figma design (as pointed out by QA check), but now looking at your screenshots, I believe it might be a good idea to be consistent and keep it as it is.

If that's the case, I apologise for not noticing. We can close this PR and move the issue to QA.

**Comment by aaemnnosttv** [comment]
Closing in favor of the current size for consistency 👍

**Pull request closed without merging**